src/pages/components/tabs/usage.mdx

```diff
@@ -187,6 +177,7 @@ B. Indicator <br /> C. Icon
 
 </Column>
 </Row>
+<Caption>Note: The icon to the left of the tab  and the secondary label is an example of a pattern. It is not currently implemented in the component.</Caption>
```

Comment:
I thought these were the features that are being added in v11? 

That is what has been in the migration docs for several months now. FYI @kingtraceyj
https://v11.carbondesignsystem.com/migrating/guide/design#tabs

Are these features going to be added soon to the carbon components or are they never going to be added?

Reply:
Yeah, echoing the similar [convo](https://ibm-studios.slack.com/archives/GSCEH4RSP/p1647981403231799) we had, I do agree we could add some clarity to the language here.

The API has been overhauled to be much more composable to enable these types of configurations. We're no longer restricting what can be put inside a tab.

For now, a few of the new tab modifiers will be something the user can configure, and we will continue to deliver more for it post-GA to provide a better experience out of the box. A style recipe or layout helper, something along those lines so users don't have to configure it all on their own.

Reply:
Just taking a shot in the dark here, something to iterate on:

```suggestion
<Caption>Note: The optional secondary label and icon are patterns enabled through the composable component API. This requires additional styling and configuration.</Caption>
```